### DISCUSSION TITLE
Praise for Markor ❤️

### DISCUSSION THREAD

**Discussion by hyprdrm** [gsantner/markor#1959] [General] [answered]
Hey, I just wanted to say this is a _great_ app - serious kudos to the devs! 👍 

I'm using it as a markdown editor, and sync'ing to my Nextcloud instance; it works really well.

The highlights for me are:
- auto-ToC
- snippets
- customisable CSS

Thank you, and please keep up the good work!

**Comment by gsantner** [comment, marked as answer]
Thanks for the warm feedback!
